native/src/main/java/io/ballerina/stdlib/io/nativeimpl/RecordChannelUtils.java

```diff
@@ -186,6 +186,8 @@ public static Object getAllRecords(BObject channel, int skipHeaders, BTypedesc t
                 }
                 Object[] out = outList.toArray();
                 return ValueCreator.createArrayValue(out, TypeCreator.createArrayType(describingType));
+            } else {
+                return IOUtils.createError("Unsupported mapping type detected. ");
```

Comment:
Only ' ' supported, but found ''
Lets format this as above

Reply:
updated in [e6b9e1a](https://github.com/ballerina-platform/module-ballerina-io/pull/898/commits/e6b9e1aed7f24cc77568f0fb702efd18da79d752)